include/rdma/fi_domain.h

```diff
@@ -249,6 +249,7 @@ enum fi_collective_op {
 	FI_REDUCE,
 	FI_SCATTER,
 	FI_GATHER,
+	FI_COLLECTIVE_OP_MAX, //must be always the last value of this enum
```

Comment:
Please do not update the include/rdma header files.  Changes there must be maintained as part of the stable ABI.  We can't add a 'max' value here, because the value is exposed to an app.  And once that value is exposed *it cannot change* without potentially breaking an app, even if a new op value is added, meaning that max is no longer the max.